app/controllers/wpcc/engine_controller.rb

```diff
@@ -22,5 +22,37 @@ def session_information
         'eligible_salary'
       )
     end
+
+    def check_wpcc_session
+      wpcc_session_expired? ? expire_wpcc_session : update_wpcc_session_expiry
+    end
+
+    def wpcc_session_expired?
+      return false unless session[:wpcc_expires_at]
+      session[:wpcc_expires_at] <= Time.current
```

Comment:
Can we change the code to: `session[:wpcc_expires_at] && session[:wpcc_expires_at] <= Time.current`?
  
            
  Write```